packages/presets/client/tests/client-preset.spec.ts

```diff
@@ -1250,7 +1250,7 @@ export * from "./gql.js";`);
   });
 
   it('should dedupe fragments in a "string" document mode', async () => {
-    const dir = path.join(__dirname, 'tmp/duplicate-fragments');
+    const dir = path.join(__dirname, 'tmp/duplicate-fragments-string');
```

Comment:
There are 2 tests in here writing to the same path, which causes a race-condition.
We can just create a separate dir for each test to avoid this issue.